packages/dm-core-plugins/src/job/JobLogsDialog.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Dialog,
+  ErrorResponse,
+  GetJobResultResponse,
+} from '@development-framework/dm-core'
+import { Button, Typography } from '@equinor/eds-core-react'
+import React from 'react'
+import hljs from 'highlight.js'
+import { FormattedLogContainer } from './JobPlugin'
+
+type AboutDialogProps = {
+  isOpen: boolean
+  setIsOpen: (newValue: boolean) => void
+  logs: string
+  error: ErrorResponse | undefined
+  result: GetJobResultResponse | null
+}
+
+export const JobLogsDialog = (props: AboutDialogProps) => {
+  const { isOpen, setIsOpen, logs, error, result } = props
+
+  return (
+    <Dialog
+      isDismissable
+      open={isOpen}
+      onClose={() => setIsOpen(false)}
+      width={'60vw'}
+      style={{ maxHeight: '70vh' }}
+    >
+      <Dialog.Header>
+        <Dialog.Title>Job logs</Dialog.Title>
+      </Dialog.Header>
+      <Dialog.CustomContent style={{ overflow: 'auto' }}>
+        <div>
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+              justifyContent: 'space-between',
+              width: '100%',
+            }}
+          >
+            <Typography variant="h6" style={{ paddingBottom: '.4rem' }}>
+              Logs:
+            </Typography>
+          </div>
+          <FormattedLogContainer style={{ maxHeight: '25rem' }}>
+            <code
+              dangerouslySetInnerHTML={
+                error
+                  ? {
+                      __html: hljs.highlight(JSON.stringify(error, null, 2), {
+                        language: 'json',
+                      }).value,
+                    }
+                  : {
+                      __html: hljs.highlightAuto(logs).value,
+                    }
+              }
+            />
+          </FormattedLogContainer>
+        </div>
```

Comment:
This could be a bad idea.
We have no controll of what's being logged. Malicous requests could be logged, and combined with this, will lead to a XSS vulnerability.

Also, logs are often not JSON, so this highlighting is not that critical, IMO :)

Reply:
indeed.
We can use `dompurify` to sanitize the logs, and prevent ( at least to a higher degree) XXS attacks

Reply:
oh, and about the hightlighting; I thought that was something that was mentioned in a meeting or something? that it was desirable to make it look slightly like the docker logs?